TodoListApp/TodoList.Api.Tests/DummyTests.cs

```diff
@@ -0,0 +1,14 @@
+﻿using NUnit.Framework;
+
+namespace TodoList.Api.Tests
+{
+    [TestFixture]
+    public class DummyTests
+    {
+        [Test]
+        public void BasicMathTest()
+        {
+            Assert.True(5 == 2 + 3);
```

Comment:
Oh, I've misled you, sorry. I mixed test frameworks and did not realize that `NUnit` does not have `Assert.True()` 😞 The constraint-based assertion model is strongly preferred nowadays and it generally makes tests more readable; please remove the https://github.com/RadekP3/kentico-onboarding-cs/pull/2/commits/f9e02ea2aa28ed71aaf67a5b5af2bd00e272aa68 commit.

Reply:
👍